__tests__/components/Send.test.js

```diff
@@ -6,6 +6,7 @@ import Send from '../../app/containers/Send/Send'
 import ZeroAssets from '../../app/components/ZeroAssets/ZeroAssets'
```

Comment:
The tests here all had to be updated because they were making assertions with the assumption that only one button existed on the page, there was also a new "panel header" button added